tools/build_config/Jenkinsfile

```diff
@@ -355,15 +372,18 @@ pipeline {
       }
       steps {
         script {
+
           if (isUnix()) {
             sh '''
+              export PATH=${PATH}:~/.local/bin &&
+              module load python/3.6 &&
```

Comment:
why is Python required for the build step? Isn't this is only required in the `Build-Docs` stage

Reply:
Again, because of the configure happening here on Windows. Also for consistency in case the analyse stage is ever skipped (which it is for Windows).